lib/README.md

```diff
@@ -138,7 +138,7 @@ which checks the dates of the control files to make sure they've been updated wh
 
 ##### openlcb.jar
  - 0.7.37 from https://repo.maven.apache.org/maven2/org/openlcb/openlcb/0.7.35/openlcb-0.7.37.jar
- - plus PRs through 265 from https://github.com/openlcb/OpenLCB_Java/pulls?q=is%3Apr+is%3Aclosed
+ - plus PRs through 266 from https://github.com/openlcb/OpenLCB_Java/pulls?q=is%3Apr+is%3Aclosed
```

Comment:
No need to change, but I suggest that you for the next PR changes `is closed` to `is merged`.

Now:
https://github.com/openlcb/OpenLCB_Java/pulls?q=is%3Apr+is%3Aclosed

Suggested change:
https://github.com/openlcb/OpenLCB_Java/pulls?q=is%3Apr+is%3Amerged

That removes PRs that are closed but not merged, for example PR https://github.com/openlcb/OpenLCB_Java/pull/263.